godtools/App/DependencyContainer/AppDataLayerDependencies.swift

```diff
@@ -266,6 +266,10 @@ class AppDataLayerDependencies {
         )
     }
     
+    func getStringSearcher() -> StringSearcher {
```

Comment:
Right I think placement could be elsewhere other than the data layer since it does seem more functionality based.
To me I feel like the domain layer is a good candidate or even in the share/common.